tests/test_external_roundtrip.rs

```diff
@@ -91,27 +116,56 @@ fn perform_test(root: &Path, prefix: &str, source: &str) {
     );
 }
 
-/// Ensure that `binjs_encode` and `binjs_decode` are built.
+/// Generate a dictionary from a source.
+///
+/// Return the path of the file containing the generated dictionary.
+fn init_dictionary(root: &Path, source: &str) -> PathBuf {
+    let root = root.join("dictionary");
+    let mut command = Command::new("target/debug/binjs_generate_prediction_tables");
+    command.arg("--in").arg(source).arg("--out").arg(&root);
```

Comment:
Maybe `.args(&[...])` would be simpler with this number of args?

Reply:
Wouldn't work for `root`, so that would give us one `args` and one `arg`, which I don't really find simpler.

Reply:
Why wouldn't?

Reply:
Ah, type mismatch.